kong/conf_loader/init.lua

```diff
@@ -772,6 +773,62 @@ local function validate_wasm(conf)
 end
 
 
+do
```

Comment:
should have `local validate_labels` before `do`

Reply:
fixed